app_grim_reefer/extra/bme280.c

```diff
@@ -0,0 +1,448 @@
+/* bme280.c - Driver for Bosch BME280 temperature and pressure sensor */
```

Comment:
Why does this file exist? The driver should be available through Zephyr

Reply:
This has to do with the unconditional 3ms delay i was talking about the other day. The driver waits everytime you read a sample even though the datasheet specifies you only need to wait after changing sensor settings. We change sensor settings during init and then the thread start delay well exceeds the needed time. The pdf i wrote up has info about what to do with it (copy it into the zephyr tree before you compile).

Reply:
For a general driver ig it makes sense since they have to deal with the possibility of people trying to read & configure at the same time but we have controlled enough circumstances I don't think we need it. If you leave it in you get significantly reduced data rate obv and it seems to helo with the i2c lockup though I never did like a statistically significant number of tests to isolate it.

Reply:
Alright